### DISCUSSION TITLE
Created a easy and composable way to configure, integrate and run readyset anywhere

### DISCUSSION THREAD

**Discussion by Peeeekay** [readysettech/readyset#48] [Show and tell]
readyset is an excellent tool that provides an innovative way of caching queries. However, I have always had trouble properly setting up and running services, whether on the cloud or on my local laptop ephemerally . To address this issue, I have developed a simple way to spin up readyset and made it composable so that it can easily integrate with the rest of my stack using [Kurtosis](https://github.com/kurtosis-tech/kurtosis).

With Kurtosis, I can run readyset with the same configuration and state anywhere. This can help with testing, prototyping, and deployment workflows to the cloud associated with readyset and services dependent on it. To demonstrate how to use the [readyset package](https://github.com/kurtosis-tech/readyset-package), I have created a small demo that automates [readyset's quickstart](https://github.com/kurtosis-tech/awesome-kurtosis/tree/main/readyset-example). If you are curious, you can check it out here and let me know what you think.

**Comment by alanarenda** [comment]
Thanks so much for sharing this @Peeeekay! We are actually working on a new quickstart using Docker Compose but it's awesome to have one using Kurtosis.
